src/scenes/home/landing/triGroup/triGroup.json

```diff
@@ -5,11 +5,12 @@
     "color": "red"
   },
   "video": {
-    "url": "https://www.youtube.com/embed/xN7yMoe38xc?modestbranding=1&start=133&showinfo=0",
+    "url":
+      "https://www.youtube.com/embed/xN7yMoe38xc?modestbranding=1&start=133&showinfo=0",
     "title": "The New Developer - Operation Code - GitHub Universe 2016"
   },
   "donate": {
-    "url": "https://opencollective.com/operationcode#support",
+    "url": "https://donorbox.org/operationcode",
```

Comment:
Does this `url` property still need to be here on the donate object? Seems like we could just use `commonUrl.donateLink` anywhere this was used.